src/amo/containers/Home.js

```diff
@@ -6,6 +7,22 @@ import translate from 'core/i18n/translate';
 
 import 'amo/css/Home.scss';
 
+const CategoryLink = ({ name, slug, text, type }) => (
```

Comment:
Also I don't think this is "complex enough to have proptypes" we just require proptypes for everything even if it isn't really useful.